Makefile

```diff
@@ -33,7 +33,7 @@ test_mock:
 	env QISKIT_TESTS=mock_online python3 -m unittest discover -s test -v
 
 test_recording:
-	rm test/cassettes/*
+	rm test/cassettes/* || true
```

Comment:
Non-blocking, [according to Make documentation](http://www2.ipp.mpg.de/~dpc/gmake/make_47.html), have you tried prepending `-` before `rm` to ignore the errors?

Reply:
much better c894e36